### PR TITLE
Adding new plugin pipeline-model-definition

### PR DESCRIPTION
**Pull request by abayer**
cc @oleg-nenashev @daniel-beck

### CODE REVIEW

**Comment by oleg-nenashev** [comment]
👎 since @abayer didn't respond to the comment in the hosting request

**Comment by abayer** [comment]
One sec...missed that.

File: permissions/plugin-pipeline-config.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: "pipeline-config"
+paths:
+- "org/jenkinsci/plugins/pipeline-config"
+developers:
+- "abayer"
+- "oleg-nenashev"
```
Comment (daniel-beck) [inline permissions/plugin-pipeline-config.yml]: Wrong user name.

**Comment by batmat** [comment]
Well, I guess this one is to be merged by @oleg-nenashev who should know quite better :-)

File: permissions/plugin-pipeline-config.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+name: "pipeline-model-definition"
+paths:
+- "org/jenkinsci/plugins/pipeline-model-definition"
```
Comment (oleg-nenashev) [inline permissions/plugin-pipeline-config.yml]: Are you sure there is no dash?

Comment (abayer) [inline permissions/plugin-pipeline-config.yml]: Yup. My instincts are always to avoid the dash in groupId. I can change it if you'd prefer.

Comment (oleg-nenashev) [inline permissions/plugin-pipeline-config.yml]: no strong opinion. We have no standard from this, and plugin devs use whatever they want

Comment (daniel-beck) [inline permissions/plugin-pipeline-config.yml]: Ideally should just be what's the default presented by `hpi:create` :-)

**Pull request merged**